common/js/src/popup-window/client.js

```diff
@@ -99,7 +100,7 @@ GSC.PopupWindow.Client.prepareAndShowAsModalDialog = function() {
   GSC.PopupWindow.Client.setWindowHeightToFitContent();
   GSC.PopupWindow.Client.setupClosingOnEscape();
   GSC.PopupWindow.Client.setupRejectionOnWindowClose();
-  GSC.PopupWindow.Client.showWindow();
+  // GSC.PopupWindow.Client.showWindow();
```

Comment:
There's a lot of code commented out in the PR, but I'm not sure it's the right approach to land it like that. Based on the previous performance, it'll probably take multiple weeks before everything gets fixed up. Also, based on discussion with Marcus yesterday, the rollout will be tricky - most likely we won't be able to roll out the extension in the next few months until sufficient number of users update to newer Chrome OS versions.

To sum up, my proposal would be to restructure the PR so that it doesn't comment out big blocks of code. Moreover, I'm proposing that we introduce two build modes: one for the App and another for the Extension, so that we still have a working App version at any point of time. Let me create a PR that would add build-time constants that we can use in the code to guard app- and extension- specific code.